lab-jaren/lib/read-many-files.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+const async = require('async');
+
+module.exports = function(filePaths, callback) {
+  var result = [];
+  async.eachOf(filePaths, function(path, index){
+    fs.readFile(path, (err, data) => {
+      if (err) return callback(err);
+      result.push(data.toString());
+      if (index === filePaths.length-1)
```

Comment:
I was trying to call the callback function with result only on the last iteration of readFile, and async's eachOf function has an index param to keep track of the index as it loops. I was thinking if filePaths.length-1 matches the index of the last path in the array, then I should invoke the callback.